packages/js/e2e-core-tests/specs/shopper/front-end-product-browse-search-sort.test.js

```diff
@@ -17,11 +17,11 @@ const {
 	beforeAll,
 } = require( '@jest/globals' );
 
-const config = require( 'config' );
+const { config } = require( '@woocommerce/e2e-environment' );
 const simpleProductName = config.get( 'products.simple.name' );
-const singleProductPrice = config.has('products.simple.price') ? config.get('products.simple.price') : '9.99';
-const singleProductPrice2 = config.has('products.simple.price') ? '1' + singleProductPrice : '19.99';
-const singleProductPrice3 = config.has('products.simple.price') ? '2' + singleProductPrice : '29.99';
+const singleProductPrice = config.get( 'products.simple.price', '9.99' );
+const singleProductPrice2 = config.get( 'products.simple.price', '19.99' );
```

Comment:
@MrJnrman Sorry, I just caught this. This suggestion applies to the next two lines because in this test we need the prices to be different. If the simple product price is configured, all three products would have the same price

```
const singleProductPrice2 = '1' +config.get( 'products.simple.price', '9.99' );
const singleProductPrice3 = '2' +config.get( 'products.simple.price', '9.99' );
```

Reply:
@rrennick Thanks for catching this. I've updated the branch